resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -66,7 +66,8 @@ ${JAVA_BIN} ${JAVA_ARGS} -Djava.security.egd=/dev/urandom \
   -m <%= EZBake::Config[:main_namespace] %> \
   --config "${CONFIG}" \
   --bootstrap-config "${BOOTSTRAP_CONFIG}" \
-  --restart-file "${restartfile}" &
+  --restart-file "${restartfile}" \
+  ${TK_ARGS} &
```

Comment:
I'm not sure if this is where it should be added...

Reply:
^ same tbh. I should up my ezbake knowledge.

Reply:
This looks good to me. Should probably be copied over to the `foreground` command as well, before the `${@}` expansion. The `stop` and `reload` commands probably don't need the extra arguments.